examples/svm/plot_rbf_parameters.py

```diff
@@ -3,48 +3,111 @@
 RBF SVM parameters
 ==================
 
-This example illustrates the effect of the parameters `gamma`
-and `C` of the rbf kernel SVM.
-
-Intuitively, the `gamma` parameter defines how far the influence
-of a single training example reaches, with low values meaning 'far'
-and high values meaning 'close'.
-The `C` parameter trades off misclassification of training examples
-against simplicity of the decision surface. A low C makes
-the decision surface smooth, while a high C aims at classifying
-all training examples correctly.
-
-Two plots are generated.  The first is a visualization of the
-decision function for a variety of parameter values, and the second
-is a heatmap of the classifier's cross-validation accuracy as
-a function of `C` and `gamma`. For this example we explore a relatively
-large grid for illustration purposes. In practice, a logarithmic
-grid from `10**-3` to `10**3` is usually sufficient.
+This example illustrates the effect of the parameters ``gamma`` and ``C`` of the
+Radius Basis Function (RBF) kernel SVM.
+
+Intuitively, the ``gamma`` parameter defines how far the influence of a single
+training example reaches, with low values meaning 'far' and high values meaning
+'close'. The ``gamma`` parameters can be seen as the inverse of the radius of
+influence of samples selected by the model as support vectors.
+
+The ``C`` parameter trades off misclassification of training examples against
+simplicity of the decision surface. A low ``C`` makes the decision surface
+smooth, while a high ``C`` aims at classifying all training examples correctly
+by give the model freedom to select more samples as support vectors.
+
+The first plot is a visualization of the decision function for a variety of
+parameter values on simplified classification problem involving only 2 input
+features and 2 possible target classes (binary classification). Note that this
+kind of plot is not possible to do for problems with more features or target
+classes.
+
+The second plot is a heatmap of the classifier's cross-validation accuracy as a
+function of ``C`` and ``gamma``. For this example we explore a relatively large
+grid for illustration purposes. In practice, a logarithmic grid from
+:math:`10^{-3}` to :math:`10^3` is usually sufficient. If the best parameters
+lie on the boundaries of the grid, it can be extended in that direction in a
+subsequent search.
+
+Note that the heat map plot has a special colorbar with a midpoint value close
+to the score values of the best performing models so as to make it easy to tell
+them appart in the blink of an eye.
+
+On both plots we see that the ``gamma`` parameter is very sensitive. If it is
```

Comment:
I'd say "The result is very sensitive to the gamma parameter" or "The decision function is very sensitive..."